### PR TITLE
refactor: no deep marshal imports

### LINKED ISSUE #6830: make swingset import E and Far from @endo/far
## What is the Problem Being Solved?

qv #3833, we should make sure that everything that needs `E` or `Far` from Endo are getting them from `import { E, Far } from '@endo/far';`, not from `@endo/marshal`, which will untangle some of our dependencies.

It looks like most everything in `packages/SwingSet/` is still importing from `@endo/marshal`.

## Description of the Design

```js
import { makeMarshal } from '@endo/marshal';
import { E, Far, passStyleOf, Remotable } from '@endo/far';
```

### LINKED ISSUE #6831: Migrate import sites for `E` and `Far` to `@endo/far`
## What is the Problem Being Solved?

Following https://github.com/Agoric/agoric-sdk/issues/3833, contracts tend to get `E` and `Far` from various places, but frequently `@endo/marshal` which entrains more components than contracts actually need and which are loosely coupled to `E` and `Far`. We should generally use the pattern `import { E, Far } from '@endo/far`.

## Description of the Design

We should do a pass over Agoric SDK and refactor dependencies such that `@endo/far` is generally the source of `E` and `Far`. Out of scope for this change would be a mechanism for injecting `endo:far` as a builtin module that execution environments with the `endo` build tag would be able to rely upon (those created by `bundleSource`) and thus not need to bundle implementations of those behaviors. Out of scope also is the conversation on whether to do that or just inject E and Far as globals.

## Scaling Considerations

This change should make contracts generally smaller and therefore cheaper to deploy.

### PR DESCRIPTION
**Pull request by erights**
Fixes #6830
or should before it is merged. This is a new requirement so please let me know of any significant impediments.
Fixes #6831
or should before it is merged. This is a new requirement so please let me know of any significant impediments.

Won't work until agoric-sdk upgraded to depend on https://github.com/endojs/endo/pull/1447

See https://github.com/endojs/endo/pull/1447 for an explanation of why this is somewhat urgent.

### CODE REVIEW

**Review by kriskowal — APPROVED** [review]
Looks mechanical to me!

File: packages/inter-protocol/src/vaultFactory/storeUtils.js [hunk 1]
```diff
@@ -5,10 +5,7 @@
 
 // XXX importing these that are declared to be used only for testing
 // until @agoric/store supports composite keys
```
Comment (turadg) [inline packages/inter-protocol/src/vaultFactory/storeUtils.js]: This comment no longer applies. It appears these functions are no longer declared to be used only for testing.
https://github.com/endojs/endo/blob/master/packages/marshal/src/encodePassable.js#L385-L389

Comment (erights) [reply packages/inter-protocol/src/vaultFactory/storeUtils.js]: Comment deleted.

**Review by turadg — APPROVED** [review]
LGTM. I see the additional commit. Best to squash since it's not relevant to the git history or changelog.

For the changelog message, this is not a change to behavior so "refactor" is more appropriate than "fix". I've taken the liberty of changing the PR title.

`automerge:squash` should turn all the changes here into one commit with the `refactor:` message.

**Comment by erights** [comment]
Closing in favor of https://github.com/Agoric/agoric-sdk/pull/7427

**Pull request closed without merging**